src/pages/cli-legacy/graphql-transformer/auth.mdx

```diff
@@ -597,7 +597,7 @@ subscription OnCreatePost {
 }
 ```
 
-Note that if your type doesn’t already have an `owner` field the Transformer will automatically add this for you. Passing in the current user can be done dynamically in your code by using [Auth.currentAuthenticatedUser()](https://docs.amplify.aws/lib/auth/manageusers/q/platform/js#retrieve-current-authenticated-user) in JavaScript, [AWSMobileClient.default().username](/sdk/auth/working-with-api.md/q/platform/ios#utility-properties) in iOS, or [AWSMobileClient.getInstance().getUsername()](/sdk/auth/working-with-api.md/q/platform/android#utility-properties) in Android.
+Note that if your type doesn’t already have an `owner` field the Transformer will automatically add this for you. Passing in the current user can be done dynamically in your code by using [Auth.currentAuthenticatedUser()](https://docs.amplify.aws/lib/auth/manageusers/q/platform/js#retrieve-current-authenticated-user) in JavaScript.
```

Comment:
Thank you for this @kuhlmanp! I think these two links might have originally been typos. They include the `.md` file extension. I think if we just remove them 

```suggestion
Note that if your type doesn’t already have an `owner` field the Transformer will automatically add this for you. Passing in the current user can be done dynamically in your code by using [Auth.currentAuthenticatedUser()](https://docs.amplify.aws/lib/auth/manageusers/q/platform/js#retrieve-current-authenticated-user) in JavaScript, [AWSMobileClient.default().username](/sdk/auth/working-with-api/q/platform/ios#utility-properties) in iOS, or [AWSMobileClient.getInstance().getUsername()](/sdk/auth/working-with-api/q/platform/android#utility-properties) in Android.
```

Reply:
I tried to find them on my own navigating those pages and via search and I couldn't find them. If you know what links they are then I am happy to put the language and links back.

Reply:
Just from what I understand, I'm not sure how these links work if I just remove the `.md` section.